unattended_installer/install_functions/indexer.sh

```diff
@@ -12,14 +12,13 @@ function indexer_configure() {
     eval "export JAVA_HOME=/usr/share/wazuh-indexer/jdk/"
 
     # Configure JVM options for Wazuh indexer
-    ram_gb=$(free -g | awk '/^Mem:/{print $2}')
```

Comment:
We need to calculate the ram using the `free` tool. If you supposed that is not necessary because the variable has a value from the specs check, it can be possible to skip the specs check with `-i` option, so we don't know what value could have.